mlflow/models/rag_signatures.py

```diff
@@ -19,7 +19,7 @@ class ChatCompletionRequest:
 
 @dataclass
 @experimental
-class MultiturnChatRequest:
+class SplitChatMessagesRequest:
     query: str = "What is mlflow?"
```

Comment:
Not related to the PR, but is there a reason we have a default for the query? 
Just for my understanding. not blocking the renaming PR

Reply:
yea we arent using this yet, but eventually we want to use the dataclass as the input example